_data/codebase/v2_3/open-source/bin-magento.yml

```diff
@@ -5107,9 +5107,11 @@ commands:
 
         <comment>bin/magento queue:consumers:start someConsumer --area-code='adminhtml'</comment>
 
-    To save PID enter path:
+    To prevent multiple copies of a consumer from running at the same time, run the consumer with the `single-thread`option: 
```

Comment:
```suggestion
    To prevent multiple copies of a consumer from running at the same time, run the consumer with the `single-thread` option: 
```

Reply:
If I'm not mistaken, changes to this file will be overwritten when we generate a new data file for the next release. Manual changes to autogenerated files is strongly discouraged for this reason. Cc: @dshevtsov for confirmation.

Reply:
@jeff-matthews, yes. 
Any such changes should be done to the corresponding codebase.
I'll add a note about this to the corresponding template.